app/src/main/kotlin/xyz/duckee/android/MainActivity.kt

```diff
@@ -43,6 +40,7 @@ import xyz.duckee.android.core.ui.LocalNavigationPopStack
 import xyz.duckee.android.core.ui.LocalPaymentSheet
 import xyz.duckee.android.core.ui.PurchaseEventManager
 import javax.inject.Inject
+import co.ab180.airbridge.Airbridge
```

Comment:
Removed Firebase Dynamic Links import and added Airbridge import. This change is necessary to replace Firebase Dynamic Links functionality with Airbridge's deeplink handling.